astro/src/content/blog/i-just-want-authentication-to-work.mdx

```diff
@@ -0,0 +1,77 @@
+---
+publish_date: 2023-11-17
+title: I Just Want Authentication To Work
+description: When I am writing an Application the last thing that I care about is setting up Authentication, I just want it to work!
+authors: Alex Patterson
+image: /img/blogs/i-just-want-authentication-to-work/i-just-want-auth-to-work.png
+categories: Company, News
```

Comment:
These probably are not correct.

Reply:
how about community

Reply:
You can look at the allowed categories here: https://github.com/FusionAuth/fusionauth-site/blob/master/.github/known-blog-categories.txt

Whatever you think fits works for me, I just was pretty sure this wasn't 'news' :)